src/App.js

```diff
@@ -2,44 +2,59 @@ import React from "react";
 import ReactDOM from "react-dom/client";
 import logo from "../images/The-Incredible-Hulk-2-icon.png"
 import userIcon from "../images/Avengers-Hulk-icon.png"
+import data from "./data.json"
+import CardComponent from "./CardComponent";
 
-/**
- * Header with nested react create elememt without using JSX
- *
- * const reactHeading = React.createElement("div", { className: "title p-3" }, [
- * React.createElement("h2", {}, "Namaste React Bootcamp"),
- * React.createElement("h3", {}, "HUSTLERS")]);
- */
 
-/**
- * Header by using react functional component
- */
-
-const TitleComponent = () => {
-  return <h1 className="title">HUSTLERS</h1>;
-};
+/** header by using react functional component */
+const title = "HUSTLERS";
 
 const SearchBar = () =>{
     return(
         <form className="search-bar">
-            <input className="input-search" type="search" placeholder="Search the Hustler.." />
+            <input className="input-search" type="search" placeholder="Search the Hustler..." />
             <button type="submit"><i className="fa fa-search fa-2x"></i></button>
         </form>
     )
 };
 
-/** Header */
+/** header component*/
```

Comment:
you can separate your components and containers in different files.
make folder structure like

/containers
   //files

/components
  //files

OR

/components
   //file
      ///file.jsx
      ///file.css